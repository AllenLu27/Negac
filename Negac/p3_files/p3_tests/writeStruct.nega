struct S {
	int a;
};

int fn(){
	struct S myS;
	output myS;
}
