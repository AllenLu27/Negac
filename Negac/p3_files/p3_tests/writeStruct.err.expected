FATAL [7,9]: Attempt to output a struct variable
Semantic analysis failed
